.travis.yml

```diff
@@ -23,21 +22,35 @@ script:
 
 matrix:
   include:
-    - env:
-        TESTNAME=quality
-        TARGETS="quality"
-    - env:
-        TESTNAME=test-python
-        TARGETS="test"
+    - python: 3.5
+      env:
+        TESTNAME=quality-python-3.5
+        TARGETS="PYTHON_ENV=py35 quality"
+    - python: 3.5
+      env:
+        TESTNAME=test-python-3.5
+        TARGETS="PYTHON_ENV=py35 test"
       after_success:
         - docker exec analytics_api_testing /edx/app/analytics_api/analytics_api/.travis/run_coverage.sh
         - codecov --disable pycov
     - env:
-        TESTNAME=test-python-django20
-        TARGETS="DJANGO_VERSION=django20 test"
-    - env:
-        TESTNAME=test-python-django21
-        TARGETS="DJANGO_VERSION=django21 test"
+        TESTNAME=test-python-3.5-django22
+        TARGETS="PYTHON_ENV=py35 DJANGO_VERSION=django22 test"
```

Comment:
I think that this is repeating:

```yaml
    - python: 3.5
      env:
        TESTNAME=test-python-3.5
        TARGETS="PYTHON_ENV=py35 test"
since django22 is the default version in the tests (See makefile default DJANGO_VERSION).

Reply:
I was thinking the same but wanted to confirm, thanks.